tests/test_name_server.rb

```diff
@@ -44,8 +44,8 @@ def no_nameserver_google
   # TESTS
 
   def test_nameserver_create_destroy_single_ipv4
-    id = '8.8.8.8'
-    refute_includes(Cisco::NameServer.nameservers, id)
+    id = '7.7.7.7'
+    assert_equal({}, Cisco::NameServer.nameservers)
```

Comment:
The argument against this change (and why I'd suggested using `includes` in the initial review instead) is the possibility that the UUT may have a default name-server configuration that already exists - unless this test is taking responsibility for removing any pre-existent config, this may fail.